packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -64,6 +64,8 @@ library TaikoData {
         uint256 ethDepositGas;
         // The maximum fee allowed for an ETH deposit.
         uint256 ethDepositMaxFee;
+        // The max number of L2 blocks that can stay unsyned on L1
+        uint8 maxUnrelayedBlocks;
```

Comment:
Perhaps something like `relayBlockThreshold` is more clear, because `maxUnrelayedBlocks` is a bit confusing because the value could be 10 but there could actually be 15 unrelayed blocks when multiple blocks are verified.